docker-compose-9.6-2.3.yml

```diff
@@ -0,0 +1,55 @@
+version: "2"
+
+volumes:
+  postgres:
+    external:
+      name: ecoroofs_postgres_data
```

Comment:
yeah. thats probably a good idea. I guess i was was originally thinking you'd just replace, but you could definitely use both.